docker/ci/install-elasticdl.bash

```diff
@@ -15,12 +15,13 @@
 
 set -e
 
-# Install ElasticDL and kubectl.
-apt-get -qq update && apt-get install -y docker.io sudo > /dev/null
-curl -sLo kubectl https://storage.googleapis.com/kubernetes-release/release/v1.14.0/bin/linux/amd64/kubectl && chmod +x kubectl && sudo mv kubectl /usr/local/bin/
+echo "Install Docker and sudo ..."
```

Comment:
I moved many of the comments into `echo` commands so that we can the progress report when the script runs.